manifests/m/mercedesclk320/SamsungRemote/1.0/mercedesclk320.SamsungRemote.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: mercedesclk320.SamsungRemote
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: mercedes.clk320
+PackageName: samsung remote control app
+License: proprietary
+ShortDescription: there are 3 files in there - an executable written in Microsoft .NET Framework 4 (you'll need .NET Fx 4 runtime), iRemote.dll which is Samsung's API and iRemoteWrapper.dll which wraps the samsung's api into .NET

```

Comment:
This is not a good short description. Please describe what the application is *used* for, not what is in the zip file

Reply:
> (you'll need .NET Fx 4 runtime)

Is this a dependency on `Microsoft.DotNet.Framework.DeveloperPack_4`?